students/JeroenSweerts/Lesson9/mailroom5.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+
+class Donor():
+    def __init__(self, name=None):
+        self._name = name
+        self._donations = []
+
+    @property
+    def name(self):
+        """Property for the donor's name."""
+        return self._name
+
+    @name.setter
+    def name(self, val):
+        """Setter for the donor's name."""
+        self._name = val
```

Comment:
These getters and setters are not necessary, since they're not doing any extra work beyond directly accessing and setting the values of the instance attributes. In Python `@propery` is generally used for calculated values, and `setter` is used where there's some extra work that needs to be done when an attribute is set.